collectors/gtfs/main.go

```diff
@@ -0,0 +1,199 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/artonge/go-gtfs"
+	"github.com/go-co-op/gocron"
+	"github.com/joho/godotenv"
+	"github.com/kiel-live/kiel-live/client"
+	"github.com/kiel-live/kiel-live/protocol"
+	log "github.com/sirupsen/logrus"
+)
+
+const IDPrefix = "gtfs-"
+
+func main() {
+	log.Infof("Kiel-Live GTFS collector version %s", "1.0.0") // TODO use proper version
+
+	err := godotenv.Load()
+	if err != nil {
+		log.Debug("No .env file found")
+	}
+
+	if os.Getenv("LOG") == "debug" {
+		log.SetLevel(log.DebugLevel)
+	}
+
+	server := os.Getenv("COLLECTOR_SERVER")
+	if server == "" {
+		log.Fatalln("Please provide a server address for the collector with COLLECTOR_SERVER")
+	}
+
+	token := os.Getenv("COLLECTOR_TOKEN")
+	if token == "" {
+		log.Fatalln("Please provide a token for the collector with MANAGER_TOKEN")
+	}
+
+	c := client.NewClient(server, client.WithAuth("collector", token))
+	err = c.Connect()
+	if err != nil {
+		log.Fatalln(err)
+		return
+	}
+	defer func() {
+		err := c.Disconnect()
+		if err != nil {
+			log.Error(err)
+		}
+	}()
+
+	g, err := loadRemoteGTFS("https://github.com/lukashass/nok-gtfs/raw/main/adler.zip")
```

Comment:
Would be nice to have this as an env var to use the collector for other gtfs feeds later on. As it seems we would have to adjust some other parts as well, maybe also sth for a followup PR.